app/models/settings.rb

```diff
@@ -2,16 +2,26 @@
 
 class Settings < ActiveRecord::Base
   class << self
-    def editing_enabled?
-      settings.editing_enabled?
+    def method_missing(name, *args, &block)
```

Comment:
I don't like overriding method_missing. Perhaps on this case we can just define the 2 methods we want:
```
def self.[](key)
   settings.data[key.to_s]
end

def self.[]=(key, value)
  settings.update data: settings.data.merge k.to_so => value
end
```
we probably can also remove the `respond_to_missing?` in this case.

Reply:
I think we can